### ISSUE TITLE
Detect if we are pod, node, bare metal, or which cloud provider

### ISSUE THREAD

**Issue by devsecfranklin** [inguardians/peirates#37]
- no need to show AWS credentials summary when not using AWS
- This dialog appears even if the cluster is not AWS/EKS

```
IAM Credentials for user AWS Credentials from Environment Variables are: 

aws_access_key_id = 
aws_secret_access_key = 
aws_session_token = 
```

**Comment by devsecfranklin** [comment]
can fix this in #38

**Comment by devsecfranklin** [comment]
We should also detect pod vs node vs bare metal since there are specific commands and results based on that situation
